src/h5web/visualizations/heatmap/store.ts

```diff
@@ -62,14 +68,12 @@ export function selectColorScale(state: HeatmapState): ColorScale {
 }
 
 export function selectDataScale(state: HeatmapState): DataScale | undefined {
-  if (state.domain === undefined) {
+  if (state.customDomain === undefined) {
     return undefined;
   }
 
   const scale = (state.hasLogScale ? scaleSymlog : scaleLinear)();
-  scale.domain(state.domain);
-  // Extend the domain to "nice" values to have nice ticks afterwards
-  scale.nice();
+  scale.domain(state.customDomain);
```

Comment:
I removed the nicing of ticks so that the values of the slider are truly the values of the dataScale.

Reply:
Right, because the "nicing" is done in the custom domain. Makes sense.